change/react-native-windows-8659aa8e-46be-4ef9-96ce-b3d5d82bc5ad.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "☠ ARM32",
```

Comment:
Can you replace this with something human readable like "Remove ARM32 support" ?
Same for the PR title so people can find it in searches in the future